ci/prow-entrypoint.sh

```diff
@@ -203,7 +252,45 @@ main () {
             cosa_build
             kola_test_metal
             ;;
-        "rhcos-90-build-test-qemu" | "rhcos-90-build-test-metal" | "scos-9-build-test-qemu" | "scos-9-build-test-metal")
+        "rhcos-90-build-test-qemu")
+            setup_user
+            cosa_init
+            # Temporary. Will be removed once variant support is in COSA
+            setup_variant "rhel-9.0"
+            cosa_build
+            # Temporarily disabled until all tests pass
+            # kola_test_qemu
+            ;;
+        "rhcos-90-build-test-metal" )
+            setup_user
+            cosa_init
+            # Temporary. Will be removed once variant support is in COSA
+            setup_variant "rhel-9.0"
+            cosa_build
+            # Temporarily disabled until all tests pass
+            # kola_test_metal
+            ;;
+        "scos-9-build-test-qemu")
+            setup_user
+            cosa_init
+            # Temporary. Will be removed once variant support is in COSA
+            setup_variant "c9s"
+            cosa_build
+            # Temporary light tests only until all tests pass
+            kola_test_qemu_light
+            # kola_test_qemu
+            ;;
+        "scos-9-build-test-metal" )
+            setup_user
+            cosa_init
+            # Temporary. Will be removed once variant support is in COSA
+            setup_variant "c9s"
+            cosa_build
+            # Temporary light tests only until all tests pass
+            kola_test_metal_light
+            # kola_test_metal
+            ;;
+        "explicitely-disabled-test")
```

Comment:
What's this about?

Reply:
It's a way to have an entry-point for Prow that works but does nothing so that we can add new tests in the Prow config without having to do changes in this repo first.

Reply:
1. Add a new test in openshift/release that calls this name
2. Make a PR that changes what this does
3. When ready, update openshift/release to get a new name
4. update your PR for the new name and merge

Without that, you have to hijack an existing test to test a new one without breaking CI for other PRs.

Reply:
Gotcha, makes sense. Might help to add a comment there explaining that. Then we can link to it too for anyone working on adding new tests.

Reply:
Thinking about this more, I've simplified it in [`a5d371c` (#905)](https://github.com/openshift/os/pull/905/commits/a5d371cf78d2923a51ef9d48597c6d91cc1099cd)